lib/ansible/modules/cloud/vmware/vmware_vspan_session.py

```diff
@@ -0,0 +1,582 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+# Copyright: (c) 2018, Ansible Project
+# Copyright: (c) 2018, CrySyS Lab <www.crysys.hu>
+# Copyright: (c) 2018, Peter Gyorgy <gyorgy.peter@edu.bme.hu>
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+
+__metaclass__ = type
+
+ANSIBLE_METADATA = {
+    'metadata_version': '1.1',
+    'status': ['preview'],
+    'supported_by': 'community'
+}
+
+DOCUMENTATION = '''
+---
+module: vmware_vspan_session
+short_description: Create or remove a Port Mirroring session.
+description:
+   - This module can be used to create, delete or edit different kind of port mirroring sessions.
+version_added: '2.8'
+author:
+- Peter Gyorgy (@gyorgypeter) <gyorgy.peter1996@gmail.com>
+notes:
+    - Tested on vSphere 6.7
+requirements:
+    - "python > =  2.6"
+    - PyVmomi
+options:
+    switch:
+        description:
+            - The name of the switch to create or remove.
```

Comment:
```suggestion
            - The name of the distributed vSwitch on which to add or remove the mirroring session.
```
(i forgot if you can create mirror sessions on normal vSwitches, if you can, it becomes `(distributed) vSwitch`)